lib/rules/jsx-no-target-blank.js

```diff
@@ -15,7 +15,7 @@ module.exports = function(context) {
         var relFound = false;
         var attrs = node.parent.attributes;
         for (var idx in attrs) {
```

Comment:
@yannickcr is there any reason this uses `for..in` instead of `Object.keys(…).forEach(…)`?